<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>813</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>819</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="ChatStackedWidget" name="GCStackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>401</width>
      <height>451</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QQListWidget" name="convationListwidget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>400</width>
        <height>400</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>400</height>
       </size>
      </property>
     </widget>
     <widget class="QWidget" name="widgetConVT" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>50</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,9">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="resource.qrc">:/resource/images/conversation.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>会话</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QQListWidget" name="applyListwidget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>400</width>
        <height>400</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>400</height>
       </size>
      </property>
     </widget>
     <widget class="QWidget" name="widgetApply" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>401</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,9">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="resource.qrc">:/resource/images/applay.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>应用</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QWidget" name="widgetFriendT" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>50</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>50</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,9">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="resource.qrc">:/resource/images/contacts.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>好友</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QQListWidget" name="friendListwidget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>400</width>
        <height>400</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>400</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QWidget" name="widgetGroupT" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>551</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,9">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="resource.qrc">:/resource/images/group.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>群组</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QQListWidget" name="groupListWidget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>400</width>
        <height>400</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>400</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="widgetBottom" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>660</y>
      <width>400</width>
      <height>120</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>120</height>
     </size>
    </property>
    <widget class="QWidget" name="widgetButtons" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-1</y>
       <width>400</width>
       <height>60</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>60</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,2,2,2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="btnFriend">
        <property name="text">
         <string>好友</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/resource/images/contacts.png</normalon>
         </iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnGroup">
        <property name="text">
         <string>群组</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/resource/images/group.png</normalon>
         </iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnConversation">
        <property name="text">
         <string>会话</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/resource/images/conversation.png</normalon>
         </iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnApply">
        <property name="text">
         <string>应用</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/resource/images/applay.png</normalon>
         </iconset>
        </property>
        <property name="popupMode">
         <enum>QToolButton::DelayedPopup</enum>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
        <property name="arrowType">
         <enum>Qt::NoArrow</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widgetMenu" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>400</width>
       <height>60</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>60</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btnButMenu">
        <property name="styleSheet">
         <string notr="true">background-color:transparent;
QPushButton#pushButton:hover{
	padding-top:2px;
	border:1px solid #FFFFFF;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/resource/common/ic_sysmen.png</normalon>
         </iconset>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnButAdd">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/resource/common/ic_user_add.png</normalon>
         </iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>354</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="btnButWea">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/resource/common/ic_weather.png</normalon>
         </iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnButCal">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/resource/common/ic_calendar.png</normalon>
         </iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="widgetUserInfo" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>400</width>
      <height>200</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>200</height>
     </size>
    </property>
    <widget class="QWidget" name="widgetWinBar" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-1</y>
       <width>400</width>
       <height>50</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>500</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,6,1,1,1">
      <property name="leftMargin">
       <number>13</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resource.qrc">:/resource/background/app.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>366</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnWinMenu">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/resource/common/ic_login_cfg.png</normalon>
         </iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnWinMin">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/resource/common/ic_min.png</normalon>
         </iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnWinClose">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/resource/common/ic_close.png</normalon>
         </iconset>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>391</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,6,1">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0,1,1">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>13</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>13</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="WidgetHead" name="widgeHead" native="true">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>80</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>80</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 2px solid black;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelUser">
           <property name="text">
            <string>用户名</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="2,3,4,2">
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="labelUserMask">
          <property name="text">
           <string>欢迎使用</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelUserName">
          <property name="text">
           <string>QuickChat</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,8">
        <item>
         <widget class="QLabel" name="labelWeather">
          <property name="text">
           <string>小雨</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChatStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header location="global">chatstackedwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QQListWidget</class>
   <extends>QWidget</extends>
   <header location="global">qqlist/qqlistwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetHead</class>
   <extends>QWidget</extends>
   <header location="global">widgethead.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
